senior_teacher.rb

```diff
@@ -1,39 +1,46 @@
-class SeniorTeacher
+require_relative 'staff'
+
+class SeniorTeacher < Staff
   attr_reader :age, :salary, :phase, :performance_rating, :target_raise
   attr_accessor :name
```

Comment:
Several of these are common to all staff members (and a few are common to students as well).  Pull them up to the appropriate superclass.